rules/zh/casual_date_test.go

```diff
@@ -10,8 +10,6 @@ import (
 )
 
 func TestCasualDate(t *testing.T) {
-	// current is Monday
-	now := time.Now()
```

Comment:
I guess this is occasional, right? If so, can you please move it back

Reply:
I have restored the changes, although with this version, my tests in zh/casual_date_test are currently not

Reply:
It seems that the reversion may have been a mistake, as my tests are not passing in zh/casual_date_test. Could we consider removing the reversion to ensure the tests function

Reply:
@rturovtsev, then ca you please remove this line then so _all_ the test are passing. 

@RexSkz, thanks for clarification, I was not too sure what is going on on that side of the project. I should have noticed this earlier, my bad.

Reply:
Thank you for your feedback! I have removed the lines that were causing the tests to fail. Now all the tests are passing.

Reply:
Hey @RexSkz, would you mind to give this PR a tick? We can't merge it without your approval.

Reply:
Sure!

Reply:
@olebedev Hi! Can you tell me if there are plans to create a new release?

Reply:
Hi there, no plans whatsoever, but I will roll this change out as a new tag soon-ish, in a matter of one or two days, bear with me.

Reply:
@rturovtsev, here you go https://github.com/olebedev/when/releases/tag/v1.1.0

Reply:
Thank you =)